features/machine/machine_misc.feature

```diff
@@ -106,3 +106,28 @@ Feature: Machine misc features testing
       | o             | json                         |
     Then the expression should be true>  @result[:parsed]['webhooks'][0]['clientConfig']['service']['port'] == 443
     """
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-37744
+  @admin
+  Scenario Outline: kube-rbac-proxy should not expose tokens, have excessive verbosity
+    Given I switch to cluster admin pseudo user
+    Then I use the "<project>" project
+
+    Given a pod becomes ready with labels:
+      | api=clusterapi| <labels>  |
```

Comment:
change this line to `| api=clusterapi | <labels> |`

Reply:
done and squashed commits